lesson05/assignment/index.html

```diff
@@ -1,14 +1,63 @@
+<!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 05</title>
-    </head>
-    <body>
-        <main class="main">
-          
-        </main>
-    </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>Assignment 05</title>
+</head>
+
+<body>
+  <main class="main">
+<!--Sets up grid for whole layout-->
+    <div class="grid-wrapper">
+<!--Sets first field in grid, used for name of firm-->
+      <div class="grid-item1">
+<!--Creates flexbox inside first field of grid-->
+        <div class="flex-item-standard">
+          <h1>design firm</h1>
+        </div>
+      </div>
+<!--Sets second field in grid, used for various links-->
+      <div class="grid-item2">
+<!--Creates flexbox inside second field of grid-->
+        <nav class="flex-item2">
+          <p>Projects</p>
```

Comment:
Typically a nav would use `<a>` for links, you can just set their href to "#" for placeholder.